lib/class_extensions.rb

```diff
@@ -0,0 +1,94 @@
+class Array
+  def my_uniq
+    arr = []
+    self.each { |num| arr << num if !arr.include?(num) }
+    p arr
+  end
+
+  def two_sum
+    arr = []
+    (0..self.length).each do |idx|
+      second_pos = idx + 1
+      (second_pos..self.length-1).each do |second_idx|
+        if self[idx] + self[second_idx] == 0
+          arr << [idx, second_idx]
+        end
+      end
+    end
+
+    p arr
+  end
+
+  def median
+    sorted = self.sort
+    if sorted.count.odd?
+      sorted[(sorted.length - 1) / 2]
```

Comment:
The floaty one is different. That is if the array count is an even number, like [1,2,3,4]. I need to do 2+3 / 2.0 to get the actual median.